universal-editor-generation-variation/src/universal-editor-ui-1/web-src/src/components/ExtensionRegistration.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * <license header>
+ */
+
+import { Text } from "@adobe/react-spectrum";
+import { register } from "@adobe/uix-guest";
+import { extensionId } from "./Constants";
+
+const generationVariationURL = 'https://experience.adobe.com/#/aem/generate-variations/';
+
+function ExtensionRegistration() {
+  const init = async () => {
+    const guestConnection = await register({
+      id: extensionId,
+      methods: {
+        headerMenu: {
+          getButtons() {
+            return [
+              // @todo YOUR HEADER BUTTONS DECLARATION SHOULD BE HERE
+              {
+                id: 'generate-variations',
+                label: 'Generate Variations',
+                icon: 'OpenIn',
+                onClick() {
+                  window.open(generationVariationURL, '_blank')
```

Comment:
@naydav Will the target `_blank` be an issue in Universal Editor? (cc @slitviachenko )

Reply:
On the latest UE main branch using Chrome I got the following error
![Screenshot 2024-02-25 at 9 19 20 PM](https://github.com/adobe/aem-uix-examples/assets/13107124/af079e82-9e6b-4576-82e2-7f0aeee71c50)

Reply:
@fe-lix- , I used a local UE setup. Didn't Chrome block pop-ups from extensions? For me it did by default, I have an option to explicitly allow pop-ups, but that is another story. I'll check with stage, prod UE then.

Reply:
I had a chat with the Editor XP team there is no pop-up protection in Universal Editor, it's a per browser configuration.